src/elements/content-sidebar/__tests__/MetadataInstanceEditor.test.tsx

```diff
@@ -63,4 +87,39 @@ describe('MetadataInstanceEditor', () => {
         const unsavedChangesModal = screen.getByText('Unsaved Changes');
         expect(unsavedChangesModal).toBeInTheDocument();
     });
+
+    test('Should call onCancel when canceling editing', () => {
+        const props: MetadataInstanceEditorProps = { ...defaultProps, template: mockCustomMetadataTemplate };
+        const { getByRole } = render(<MetadataInstanceEditor {...props} />);
+
+        act(() => {
+            fireEvent.click(getByRole('button', { name: 'Cancel' }));
```

Comment:
```suggestion
            await userEvent.click(getByRole('button', { name: 'Cancel' }));
```
I dont think we need `act` if we are using the right combination of functions, if you still get the `act` error/warning still i wondering its you need to do use `findByRole` instead of `getByRole`which also might fix that error/warning

Reply:
I changed that to not use it, is `act` deprecated or unwanted here though ? I thought it's just recommended to change [where](https://react.dev/warnings/react-dom-test-utils) it's imported from.